lib/app_profiler/viewer/base_viewer.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+gem "rails-html-sanitizer", ">= 1.6.0"
+require "rails-html-sanitizer"
```

Comment:
This would require folks using the basic speedscope viewer to require rails-html-sanitizer, but it isn't actually needed. I think this is the wrong place for this. We can make a subclass for remote profile viewing instead.

Reply:
This is currently not in the right place. We should only make those interested in remote viewers have this as a dependency. Placing it in base viewer will require it for everyone, right?

Reply:
Moved this in ad67c69 to not be part of the base viewer.